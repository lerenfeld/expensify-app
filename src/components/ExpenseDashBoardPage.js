import React from 'react';
import ExpenseList from './ExpenseList';
import ExpensesSummary from './ExpensesSummary';



const ExpenseDashBoardPage = () => (
  <div>
    <ExpensesSummary/>
    <ExpenseList />
  </div>
)

export default ExpenseDashBoardPage;