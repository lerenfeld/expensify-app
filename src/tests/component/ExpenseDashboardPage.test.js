
import React from 'react'
import ExpenseDashBoardPage from '../../components/ExpenseDashBoardPage'
import {shallow} from 'enzyme'

test('should render ExpenseDashBoardPage correctly',  ()=>{
  const wrapper = shallow(<ExpenseDashBoardPage/>)

  // const renderer = new ReactshallowRenderer 
  // renderer.render(<Header />)
  // expect(renderer.getRenderOutput()).toMatchSnapshot()

  // expect(wrapper.find('h1').text()).toBe('Expensify');

  expect(wrapper).toMatchSnapshot();
});
