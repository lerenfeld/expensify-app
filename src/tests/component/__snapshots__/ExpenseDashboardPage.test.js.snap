// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashBoardPage correctly 1`] = `
<div>
  <React.Memo />
  <React.Memo />
</div>
`;
